### DISCUSSION TITLE
unable to update to 2.7.0

### DISCUSSION THREAD

**Discussion by kaunix** [mrlt8/docker-wyze-bridge#1100] [Q&A]
ran docker-compose stop, rm, pull, up -d, docker image prune. relogged in and authed. and it still says 2.6.0. did it multiple times and got the same thing...

any ideas?

**Comment by AngusNB** [comment]
# it still says 2.6.0

Are you referring to the WebUI? If so that seems to be a bug.

**Comment by kaunix** [comment]
YesOn Jan 20, 2024, at 09:25, AngusNB ***@***.***> wrote:﻿
it still says 2.6.0
Are you referring to the WebUI? If so that seems to be a bug.
